components/src/atoms/information/Information.html

```diff
@@ -0,0 +1,11 @@
+<div class="information">
+  <a-heading
+    :level="level"
+    class="information__title"
+  >
+    {{ title }}
+  </a-heading>
+  <a-paragraph class="information__text">
+    {{ text }}
+  </a-paragraph>
+</div>
```

Comment:
This component should not be here, please move to previous place, in the future we will move it to cookbook